Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
and I see you are adding parentheses around all macros everywhere...my focus is to make sure to have parentheses where the macro is defined like `#define NAME (with + in + parentheses)` (and needed, doing it for plain numbers are not needed. By doing that one can later use `NAME` everywhere without needing parentheses (because they are already there). I like LISP, but by doing like this one do not have to have extra parentheses everywhere and you have less clutter (easier reading)